backend/app/services/streaming/core.py

```
@@ -613,6 +613,48 @@ async def finalize(self) -> dict[str, Any]:
+                                assistant_message=self.state.full_response,
+                                is_group_chat=self.state.is_group_chat,
+                                sender_name=(
+                                    self.state.user_name
+                                    if self.state.is_group_chat
+                                    else None
+                                ),
+                            )
+                finally:
+                    db.close()
+        except Exception:
+            pass  # Never block on memory
```

Ruff (0.14.11)

`try`-`except`-`pass` detected, consider logging the exception

(S110)